js/engine.js

```diff
@@ -131,7 +131,7 @@ var Engine = (function(global) {
                  * so that we get the benefits of caching these images, since
                  * we're using them over and over.
                  */
-                ctx.drawImage(Resources.get(rowImages[row]), col * 101, row * 83);
+                ctx.drawImage(Resources.get(rowImages[row]), col * 101, row * 85);
```

Comment:
can you create constants with the values 101 and 85, with the name of them we can place a context around these numbers.